services/QuillLMS/client/app/bundles/Shared/libs/commaSeparatedValuesString.ts

```diff
@@ -0,0 +1,12 @@
+export function commaSeparatedValuesString(values: string[]) {
+  if(!values.length) { return null }
+  let valuesString = '';
+  values.forEach((value, i) => {
+    if(i === values.length - 1) {
+      valuesString += `${value}`;
+    } else {
+      valuesString += `${value}, `;
+    }
+  });
+  return valuesString;
+}
```

Comment:
Couldn't you just do this with `.join(', ')`?

Reply:
yeah, originally I was going to do some other formatting but this can def be removed in favor of just using join